src/components/ebay-carousel/test/test.browser.js

```diff
@@ -3,10 +3,14 @@ const expect = require('chai').expect;
 const testUtils = require('../../../common/test-utils/browser');
 const mock = require('../mock');
 const renderer = require('../');
-
-const privates = renderer.privates;
-const constants = privates.constants;
-const containerWidth = 800; // puppeteer default
+const styleOverrides = document.createElement('style');
+// No need to run the full length animation when testing the carousel.
+styleOverrides.innerHTML = `
+    .carousel__list {
+        transition: transform 0.1s linear;
```

Comment:
Should we disable transition altogether here? Or if we do need to keep it, could we make it even shorter? By the way, maybe make this a one-liner since intent+comment is very clear.

Reply:
Not against making it a one liner, but then the linter would not be happy. Also the reason I don't have it set to be a 0s transition is because then the `transitionend` events won't fire.

Reply:
Ah, right. Maybe we could shorten it to 0.05s or less? It's not super noticeable in this case, but does add up. (I guess it's more noticeable in the `dialog`).

Reply:
Yeah I can decrease it more. I plan on creating another PR that does the same thing for the dialog tests.